### DISCUSSION TITLE
How to adjust width to edges for reader on tablets in landscape for one column mode?

### DISCUSSION THREAD

**Discussion by andrewlauNYPL** [readium/kotlin-toolkit#522] [Q&A] [answered]
Is there a way to adjust have the width of the publication to extend to the edges for the Ereader for tablet in landscape in single column? There are no issues with the reader reaching the edges for landscape on mobile devices and tablet in portrait.

Tried adjusting the `pageMargins`, `RsProperties(colWidth, maxLineLength)` 

Thanks!

**Comment by mickael-menu** [comment, marked as answer]
It's not possible right now, but will be in a future version of Readium CSS, which the Kotlin toolkit relies on.

See https://github.com/readium/readium-css/issues/130

**Reply by andrewlauNYPL** [reply]
Ah all good, thanks!
